src/transformers/__init__.py

```diff
@@ -1442,6 +1443,15 @@
 
     # PyTorch models structure
 
+    _import_structure["models.dbrx"].extend(
```

Comment:
Again, add `"DBRX_PRETRAINED_CONFIG_ARCHIVE_MAP"` here too. This is what all the other models have.

Reply:
Hey! As of a [very recent PR](https://github.com/huggingface/transformers/pull/29112), the pretrained checkpoint maps are no longer required, so you can just remove these.